testsuite/integration-arquillian/tests/other/console/pom.xml

```diff
@@ -106,6 +106,7 @@
                     <configuration>
                         <systemProperties>
                             <keycloak.theme.dir>${keycloak.theme.dir}</keycloak.theme.dir>
+                            <keycloak.profile.feature.account2>enabled</keycloak.profile.feature.account2>
```

Comment:
I believe we really shouldn't hardcode the profile here. This way we'd run all Admin Console tests with a preview profile – always, even when e.g. testing product.

I've been a bit playing with this and I think we should use `@EnableFeature` annotation for a separate test just for the new switch. You can use my changes if you want: https://github.com/ssilvert/keycloak/pull/4

Reply:
I agree.  Didn't know about the @EnableFeature annotation.  That's much better.